packages/edit-post/src/components/sidebar/settings-sidebar/style.scss

```diff
@@ -1,12 +1,13 @@
 .edit-post-settings-sidebar__panel-block .components-panel__body {
 	border: none;
 	border-top: $border-width solid $light-gray-500;
-	margin: 0 -16px;
+	margin: 0 -$grid-size;
 
 	.components-base-control {
-		margin: 0 0 1.5em 0;
+		margin-bottom: $grid-size;
```

Comment:
The 1.5em translate to 19.5px, which feels pretty good. What if we did `$grid-size * 3` here instead? It would come to 24px but that seems okay to me? Here's 24px:

<img width="309" alt="screenshot 2019-02-01 at 10 13 55" src="https://user-images.githubusercontent.com/1204802/52113611-1c6dd800-260a-11e9-98a1-f488348c4ffb.png">

At the very least, it should be `$grid-size-large`.